pages/index.vue

```diff
@@ -254,6 +250,46 @@ export default class extends QiskitPage {
 @import '~carbon-components/scss/globals/scss/typography';
 
 .landing-page {
+  background-color: $white-background-ui;
+
+  &__presentation-container {
+    position: relative;
+    padding-top: $layout-05;
+    padding-bottom: $layout-05;
+    background-attachment: fixed;
+    background-size:
+      2rem 2rem,
+      2rem 2rem;
+    background-image:
+      linear-gradient(to right, $cool-gray-10 1px, transparent 1px),
+      linear-gradient(to bottom, $cool-gray-10 1px, transparent 1px);
+    background-position:
+      -1rem center,
+      -1rem center;
+    background-repeat:
+      repeat,
+      repeat;
+
+    overflow: hidden;
+  }
+
+  &__presentation-picture {
+    position: absolute;
+    width: 100%;
+    top: 0;
+    bottom: 0;
+    left: 40%;
+    left: unquote("max(40%, #{$small})");
```

Comment:
`left` attribute is set twice so, the first one is not used. We should have only one

Reply:
CSS function `max` is not extensively supported yet. The browsers not supporting it will drop the rule and resort to the previous value. I'll add a comment to clarify.

Reply:
Fortunately, we've been able to remove this and use a simple positioning scheme.